lte/gateway/deploy/roles/magma_deb/files/post_install_changes.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
```

Comment:
This file will be migrated to ansible - this is ongoing and will be pushed tomorrow.